lib/printers/junit.js

```diff
@@ -0,0 +1,26 @@
+const Linter = require('../index');
```

Comment:
Let's remove this one for now? It doesn't seem correct (looks like a duplicate of the JsonPrinter), and maybe still WIP.

Reply:
whooops, that shouldn't have ended up here 😱